### PR TITLE
Standardise font sizes

### PR DESCRIPTION
**Pull request by jkempster34**
### Trello card
https://trello.com/c/IGJu2Apb

### Context
Standardise font sizes. This also improves accessibility because users can change the font size in their browsers.

### Changes proposed in this pull request
- Update font size to use rem constants
- Remove unused CSS

### Guidance to review
I have tried to reflect the current font sizes exactly. 
Removed some small bits of CSS that aren't used.
Also had a browse with font sizes set to extra-large and all seems fine. Image below shows after and before with browser font size set to extra large.
image

### CODE REVIEW

File: app/webpacker/styles/content.scss [hunk 1]
```diff
@@ -18,15 +18,9 @@
         vertical-align: top;
         width: 66.6666%;
         float: left;
```
Comment (jkempster34) [inline app/webpacker/styles/content.scss:20]: Removed CSS here

File: app/webpacker/styles/components/event-box.scss [hunk 2]
```diff
@@ -1,16 +1,3 @@
-.event-link {
```
Comment (jkempster34) [inline app/webpacker/styles/components/event-box.scss:1]: And here

File: app/webpacker/styles/components/event-box.scss [hunk 3]
```diff
@@ -1,16 +1,3 @@
...
-    &:focus {
-        outline: none;
```
Comment (jkempster34) [inline app/webpacker/styles/components/event-box.scss:3]: Removed CSS here

File: app/webpacker/styles/content.scss [hunk 4]
```diff
@@ -18,15 +18,9 @@
...
-        .jumbotron {
-            font-size: 19px;
```
Comment (jkempster34) [inline app/webpacker/styles/content.scss:22]: And here

File: app/webpacker/styles/components/event-box.scss [hunk 5]
```diff
@@ -1,16 +1,3 @@
...
-        border: none;
-        background: transparent;
-        box-shadow: none;
-    }
-}
-
-.events-listing p {
-  font-size: 300px;
```
Comment (jkempster34) [inline app/webpacker/styles/components/event-box.scss:11]: And here

**Review by peteryates — APPROVED** [review]
Looks good to me, had a look around the site and everything feels like it's working well.

We have an awful lot of font sizes, I think a good follow up might be to try to reduce the number, but this is a huge step in laying the groundwork for that 👍🏽

**Pull request merged**